api/nakadi-event-bus-api.yaml

```diff
@@ -1151,6 +1263,36 @@ paths:
 # ################################### #
 
 definitions:
+  Storage:
+      type: object
+      description: |
+        This is a storage backend
+        A real one.
+      properties:
+        id:
+          description: The ID of the storage backend
+          type: string
+        storage_type:
+          description: the type of storage (currently, only Kafka)
+          type: string
```

Comment:
@v-stepanov made a good point below about enums not being as easily extensible, so I went back to String, with the permitted value in the description: fe7b7b9